EazyE2E/ElementHelper/EzTextResult.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace EazyE2E.ElementHelper
+{
+    public class EzTextResult<T>
+    {
+        public bool NotSupported { get; set; }
+        public bool IsMixed { get; set; }
+        public T Result { get; set; }
```

Comment:
For more details, see Trello ticket https://trello.com/c/RyGDk3Mz/32-add-a-lazy-load-property-on-ezelement-to-get-al-lthe-underlying-text-properties-this-should-throw-an-exception-of-there-is-no-un